projects/components/src/table/table.component.scss

```diff
@@ -49,9 +49,9 @@
     border-top: 1px solid $blue-2;
 
     &.hovered-row {
-      background: $blue-2;
```

Comment:
This is for hover on a selected row. Has the ux changed for this? At all other place, we make the color darker on hover. This would make it inconsistent with similar behavior on a card list

Reply:
Yeah. So hover on these rows right now is a little off. We go darker on buttons typically, but with rows and other navigation items the hover I think would be better suited to always be the same. 

Alternatively, I could accept that a hover on a selected row doesn't change anything. Instead it stays as a selected state.

Reply:
I am fine with whatever you decide on visual aspects. I just want to share that this change require similar changes to
- Sequence chart: d3 row hover. Without this waterfall would appear broken.
- Card lists

@michael-traceable Could you confirm we need to apply styling at these two locations too?

Reply:
Good call! Yes, waterfall should match because it's similar structure (table format)... The card list we can match as well, however, I believe I also have a ticket that the hover on that should just be the shadow and no gray, but it doesn't bother me too much.

Reply:
Should we bundle all of these changes at once? I kind of lean towards yes so there are no inconsistencies.

Reply:
I agree. @palbizu Can we bundle all the changes together in the same PR?

Reply:
@anandtiwary just to double-check. I should apply the same pattern to
1- Tables (I already did it in this PR)

Reply:
Yes... but with 3, please see the intended hover for a card item. https://www.figma.com/file/tVP4INp6sTDVPx6H7y5efk/%F0%9F%93%90-Components?node-id=2841%3A1

![image](https://user-images.githubusercontent.com/80359572/117029049-867af900-acbb-11eb-898e-8dad9a4effc3.png)